test/server/cards/03_TWI/units/BattleDroidLegion.spec.ts

```diff
@@ -0,0 +1,33 @@
+describe('Battle Droid Legion', function() {
+    integration(function(contextRef) {
+        describe('Battle Droid Legion\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['power-of-the-dark-side']
+
+                    },
+                    player2: {
+                        groundArena: ['battle-droid-legion']
+                    }
+                });
+            });
+
+            it('should create 3 Battle Droid tokens when defeated', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.powerOfTheDarkSide);
+                expect(context.player2).toBeAbleToSelectExactly([context.battleDroidLegion]);
```

Comment:
Don't need this expectation since we're not testing Power of the Dark Side. Same for the defeat check on line 24

Reply:
fixed up